resources/roadmap/index.md

```diff
@@ -239,9 +239,11 @@ meta_descr: meta_descr.roadmap
                                 <h3 class="months">{% t roadmap.comingsoon %}</h3>
                                     <li class="ongoing"><a href="https://github.com/Rucknium/OSPEAD">{% t roadmap.ospead %}</a></li>
                                     <li class="ongoing">{% t roadmap.bp++ %}</li>
-                                    <li class="ongoing">{% t roadmap.cuprate %}</li>
+                                    <li class="ongoing"><a href="https://cuprate.org/">{% t roadmap.cuprate %}</a></li>
+                                    <li class="ongoing"><a href="https://github.com/seraphis-migration/strategy/wiki/Our-Wallet-Rewrite-Project-After-Priority-Switched-From-Seraphis-to-FCMPs">{% t roadmap.fcmp-plus %}</a></li>
+                                    <li class="upcoming">BasicSwapDEX</li>
+                                    <li class="upcoming">SeraiDEX</li>
```

Comment:
NACK.

This is a massive overreach.

I made the decision for Serai to be a freestanding project from Monero years ago and I've stuck by it since, not accepting any CCS funding. I even wrote an entire blog post going over how we're a project integrating Monero, not a Monero project. This is actively against Serai's position and will create harmful misrepresentations of what Serai is.

If anyone now wants to claim my work under their banner, they can start by offering a donation of all costs I've accrued during my work.

---

As this'll inevitably come up, I am considering CCS funding the audit of monero-serai as a public good to the Monero community. That is not Serai, it is a library built and used by Serai. If at that time, the Monero site wants to advertise their auditing of monero-serai, I'd have no objections.

---

"Integration by Serai DEX"

That would be fine with me. It is upcoming, it is related to Monero, and clarifies the work was done on Serai's end.